src/components/run-button.js

```diff
@@ -121,27 +144,32 @@ const RunButton = (props) => {
         return props.getStatus(getRunnable());
     }
 
-    const runningStatus = getRunningStatus();
```

Comment:
why remove this ?

Reply:
defining this variable is not really useful